task/handler.go

```diff
@@ -47,7 +47,7 @@ func KoboldHandler(ctx context.Context, cache string, g model.TaskGroup, runner
 
 	msg, err = GetCommitMessage(changes)
 	if err != nil {
-		msg = "chore(kobold): Update image refs"
+		return nil, fmt.Errorf("unable to get commit message: %w", err)
```

Comment:
@rossigee , what error are we handling here? The function doesnt even return an error. Well it returns always nil so I under understan why its even returning an error.

You have to check the error of fmt.Sprintf I think. This one may error but we are not checking it, so you would get an empty string

Reply:
Haha. Good spot. It isn't actually raising any errors in it's current implementation :roll_eyes:

We could take it out, but I guess it'd only end up going back in again (if/)when the ability to template the message becomes available again.

Reply:
for the record. The error is checked in this PR.